app/models/product.rb

```diff
@@ -0,0 +1,4 @@
+class Product < ApplicationRecord
+  validates :name, presence: true
+  validates :price, presence: true
```

Comment:
Also add a validation of price being a number and always positive.

Reply:
done.